boards/arm/ubx_evknorab10_nrf5340/Kconfig.defconfig

```diff
@@ -0,0 +1,87 @@
+# u-blox EVK-NORA-B10 board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+if  BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP || BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP_NS
+
+config BOARD
+	default "ubx_evknorab10_nrf5340_cpuapp" if BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP || BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP_NS
```

Comment:
the `if` clause here is redundant to line 6

Reply:
I was using the nrf5340dk_nrf5340 as a template - same lines are there.